src/main/java/Duke.java

```diff
@@ -57,22 +61,34 @@ public static void respond(String input, List<Task> tasks) {
             System.out.println("OOPS!!! " + e.getMessage());
         }
     }
+
+    public static void delete(int taskNumber, List<Task> tasks) {
+        int index = taskNumber - 1;
+        Task removedTask = tasks.remove(index);
+        System.out.println("Noted. I've removed this task:\n"
+                + removedTask + "\n"
+                + "Now you have " + tasks.size() + " tasks in the list.");
+    }
+
+    // combines a String array from index 1 to (length - 1)
     public static String combine(String[] parts) {
         String result = parts[1];
         for (int i = 2; i < parts.length; i++) {
             result += " " + parts[i];
         }
         return result;
     }
+
     // Mark a task as done by substituting the current task with a completed task in the list
     public static void markAsDone(int taskNumber, List<Task> tasks) {
         // the index of a task in the list is one less than the taskNumber
         int index = taskNumber - 1;
```

Comment:
Could just minus 1 when invoking the method. Rather than creating a new variable.